scripts/xtensa-build-zephyr.py

```diff
@@ -559,7 +581,13 @@ def main():
 	else:
 		print("Building platforms: {}".format(" ".join(args.platforms)))
 
-	west_init()
+	west_root_dir, west_manifest_path = find_west_workspace()
+	if not west_root_dir:
+		execute_command(["west", "init", "-l", f"{SOF_TOP}"], cwd=west_top)
+	else:
+		print(f"West workspace: {west_root_dir}")
```

Comment:
This part of the code is short yet quite confusing because `find_west_workspace()` still has its old name of a function without any side-effect but made a major change to it: now it can invoke a duplicate `west (re-)init` execute_command() buried inside `west_reinitialize()`.

The `else:` clause does not help: why are the values not logged after this `west init`?

I think the smallest "fix" is to correct the name of `find_west_workspace()` to something like  `west_init_if_needed()` and have all `west init` commands there.

Then log the values always here.